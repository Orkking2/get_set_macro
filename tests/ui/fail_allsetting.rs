use get_set_macro::get_set;

#[get_set(get)]
struct Example {
    // Will not have any getters or setters generated
    #[gsflags(skip)]
    skipped: f32,
}

fn main() {
    let mut example = Example {
        skipped: 1.0,
    };

    // Should be no method named `get_skipped` generated for `Example`.
    example.get_skipped();
}