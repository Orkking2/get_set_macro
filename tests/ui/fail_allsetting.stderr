error[E0599]: no method named `get_skipped` found for struct `Example` in the current scope
  --> tests/ui/fail_allsetting.rs:15:13
   |
4  | struct Example {
   | -------------- method `get_skipped` not found for this struct
...
15 |     example.get_skipped();
   |             ^^^^^^^^^^^ method not found in `Example`
